agent/sidecar_service.go

```diff
@@ -202,14 +213,11 @@ func (a *Agent) sidecarPortFromServiceID(sidecarCompoundServiceID structs.Servic
 	return sidecarPort, nil
 }
 
-func sidecarDefaultChecks(serviceID string, localServiceAddress string, port int) []*structs.CheckType {
```

Comment:
I removed the comments here because the first one:

```// Setup default check if none given```

Is already commented where this function is called and the second one is now commented where we pass in the address.